cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java

```diff
@@ -411,45 +407,6 @@ protected List<EnvVar> getEnvVars() {
         varList.add(ContainerUtils.createEnvVar(ENV_VAR_STRIMZI_GC_LOG_ENABLED, String.valueOf(gcLoggingEnabled)));
         JvmOptionUtils.javaOptions(varList, jvmOptions);
 
-        varList.add(ContainerUtils.createEnvVar(ENV_VAR_KAFKA_BRIDGE_BOOTSTRAP_SERVERS, bootstrapServers));
-        varList.add(ContainerUtils.createEnvVar(ENV_VAR_KAFKA_BRIDGE_ADMIN_CLIENT_CONFIG, kafkaBridgeAdminClient == null ? "" : new KafkaBridgeAdminClientConfiguration(reconciliation, kafkaBridgeAdminClient.getConfig().entrySet()).getConfiguration()));
-        varList.add(ContainerUtils.createEnvVar(ENV_VAR_KAFKA_BRIDGE_ID, cluster));
-
-        if (kafkaBridgeConsumer != null) {
-            varList.add(ContainerUtils.createEnvVar(ENV_VAR_KAFKA_BRIDGE_CONSUMER_CONFIG, new KafkaBridgeConsumerConfiguration(reconciliation, kafkaBridgeConsumer.getConfig().entrySet()).getConfiguration()));
```

Comment:
Can KafkaBridgeConsumerConfiguration be deleted since it was only used here? The same goes for KafkaBridgeProducerConfiguration.

Reply:
That seems wrong. That is how user passes the configuration. Those classes should be used.

Reply:
Oh I see, I was wondering where these model classes are used but these are used by the users. Please ignore this comment.

Reply:
No, what I meant is that I think they need to be used in this PR -> the process the user configuration from the KafkaBridge CR and should be used to pass it into the config file here. What I was trying to say is that this likely means some bug that should be fixed rather than that the files should be deelted.

Reply:
Ok, so that's what I was wondering in the first place. Perhaps these should be used in the config builder class similar to https://github.com/strimzi/strimzi-kafka-operator/blob/c1b20f726dddbcd2a070c2eeb14fd30902027aec/cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java#L478?

Reply:
Yeah. Or maybe this is more corresponding: https://github.com/strimzi/strimzi-kafka-operator/blob/c1b20f726dddbcd2a070c2eeb14fd30902027aec/cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java#L795

Reply:
That should not work that way. The configuration classes are what ensures the filtering of forbidden options, defaults etc. They are used everywhere else and I think we should stick with that.

Reply:
You are right. I will fix this.